templates/_linux.yaml

```diff
@@ -18,6 +18,10 @@
 {% for osl in oslabels %}
     name.os.template.kubevirt.io/{{ osl }}: {{ lookup('osinfo', oslabels[0]).name }} or higher
 {% endfor %}
+{% for osl in oslabels %}
+    datavolume.template.kubevirt.io/{{ osl }}: "{{ osl }}"
```

Comment:
Maybe use the annotations `pvc.template.kubevirt.io/name` and `pvc.template.kubevirt.io/namespace` to make it explicit that the annotations point to a PVC? Some might think they point to a DV (I know we already agreed on `datavolume` before sorry about that :) )